utils/kafka-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaConfig.scala

```diff
@@ -48,7 +48,7 @@ object KafkaConfig {
   import net.ceedubs.ficus.readers.EnumerationReader._
   import TopicsExistenceValidationConfig._
 
-  val lowLevelComponentsEnabled: Boolean = true
+  val lowLevelComponentsEnabled: Boolean = false
```

Comment:
Let's change changelog/migrationguide, as currently it describes this flag as true by default?

Reply:
yes yes, I remember